s2fft/transforms/spherical.py

```diff
@@ -537,7 +542,7 @@ def forward_numpy(
     return flm * (-1) ** spin
 
 
-@partial(jit, static_argnums=(1, 3, 4, 5, 7, 8))
+@partial(jit, static_argnums=(1, 3, 4, 5, 7, 8,9))
```

Comment:
```suggestion
@partial(jit, static_argnums=(1, 3, 4, 5, 7, 8, 9))
```

Reply:
Didn't want to format and ruin the entire file :/